src/OpenTK/Graphics/GraphicsContext.cs

```diff
@@ -112,6 +129,18 @@ public GraphicsContext(GraphicsMode mode, IWindowInfo window, int major, int min
                 if (minor < 0)
                     minor = 0;
 
+                // Angle needs an embedded context
+                var use_angle_flag = GraphicsContextFlags.Angle
+                                | GraphicsContextFlags.AngleD3D9
+                                | GraphicsContextFlags.AngleD3D11
+                                | GraphicsContextFlags.AngleOpenGL;
+                var use_angle = false;
+                if ((flags & use_angle_flag) != 0)
```

Comment:
I don't have any issues with changing the naming convention for the code I introduced but I just wanted to make you aware that there are a lot of pre-existing variable names which are named according to the `snake_style ` convention, even just in the constructor in GraphicsContext.cs which you commented on above (i.e. `handle_cached`). In fact there seems to be a discrepancy as I see some locally defined variable names using `camelCase` whereas a lot of member variable names are `snake_style`. Is this a rule?